cypress/integration/portal-dashboard/portal-dashboard-feedback.spec.js

```diff
@@ -65,7 +65,7 @@ context("Portal Dashboard Feedback Panel", () => {
       cy.get('[data-cy=class-nav]').should('be.visible');
       cy.get('[data-cy="num-Awaiting feedback"]').should('be.visible');
       cy.get('[data-cy=item-number').should('contain', '3');
-      cy.get('[data-cy=sort-feedback]').should('be.visible');
+      cy.get('[data-cy=sort-feedbackreport]').should('be.visible');
```

Comment:
Previously we were setting this to `sort-feedback` with a static value in `PopupClassNav`. The updated code sets the value dynamically based on `viewMode`. I didn't think it was worth trying to match the old value.